handle_actions.go

```diff
@@ -18,50 +19,57 @@ func (ms MocksArray) DoActions(c Context) error {
 	return nil
 }
 
+// DoActions checks if a given condition matches and performs each action in m.Actions
 func (m *Mock) DoActions(c Context) error {
 	c.Values = m.Values
 	if !c.MatchCondition(m.Expect.Condition) {
 		return nil
 	}
+
 	for _, actionDispatcher := range m.Actions {
 		actualAction := getActualAction(actionDispatcher)
-		if err := actualAction.Perform(c); err != nil {
-			logrus.WithFields(logrus.Fields{
-				"err":    err,
-				"action": fmt.Sprintf("%T", actualAction),
-			}).Errorf("failed to do action")
-		}
+		actualAction.Perform(c)
 	}
+
 	return nil
 }
 
-func (a ActionSendHTTP) Perform(context Context) error {
-	bodyStr, err := context.Render(a.Body)
-	if err != nil {
-		return err
-	}
+// Perform defines the action to perform for the ActionSendHTTP action
+func (a ActionSendHTTP) Perform(context Context) {
+	go func() {
+		if a.Sleep != 0 {
+			time.Sleep(a.Sleep)
+		}
```

Comment:
We should be able to remove the conditional because `time.Sleep(0)` should return immediately. https://golang.org/pkg/time/#Sleep